docs/docs/learn/architecture.mdx

```diff
@@ -52,9 +57,9 @@ graph TB
     Server2["MCP Server 2<br/>(e.g., Filesystem)"]
     Server3["MCP Server 3<br/>(e.g., Database)"]
 
-    Client1 ---|"One-to-one<br/>connection"| Server1
-    Client2 ---|"One-to-one<br/>connection"| Server2
-    Client3 ---|"One-to-one<br/>connection"| Server3
+    Client1 ---|"Many-to-one<br/>relationship"| Server1
+    Client2 ---|"One-to-one<br/>relationship"| Server2
+    Client3 ---|"One-to-one<br/>relationship"| Server3
```

Comment:
I don't think this makes it clearer.  The Sentry server may be serving multiple clients, but the connection between **Client 1** and Sentry is still one-to-one.  (And I don't think changing "connection" to "relationship" helps.)

How about adding another client?

```suggestion
    Client1 ---|"One-to-one<br/>connection"| Server1
    Client2 ---|"One-to-one<br/>connection"| Server1
    Client3 ---|"One-to-one<br/>connection"| Server2
    Client4 ---|"One-to-one<br/>connection"| Server3
```

(Will also need to add `Client4["MCP Client 4"]` above this.)

Possibly also name the servers as "Server [A-C]", instead of "Server [1-3]".

Reply:
yes. How about we just do away with 1-1 as it seems to be causing the confusion and just explicitly say as below ?
```
Client1 ---- "dedicated connection" | ServerA
Client2 ---- "dedicated connection" | ServerB
Client3 ---- "dedicated connection" | ServerC
Client4 ---- "dedicated connection" | ServerC
```
An alternative for "dedicated connection" could be "direct connection"

Reply:
Sure. Making this change.  Here's how it looks right now..
<img width="757" height="242" alt="image" src="https://github.com/user-attachments/assets/b499feba-a2a4-489a-b208-9465cdbfa4c7" />

BTW, I am not entirely sure if the same MCP host can have 2 clients for the same remote server (as in Client3 and Client4 both connecting to Sentry remote server) - likely yes if seperate conversation tabs have seperate clients - but just bringing it up here to be sure.  Something like below..

<img width="747" height="235" alt="image" src="https://github.com/user-attachments/assets/fff1a18b-cd67-4f93-a557-380393f6b7bb" />